packages/components/message/index.ts

```diff
@@ -1,8 +1,8 @@
-import { withInstallFunction } from '@element-plus/utils/with-install'
+import { withFunctionInstall } from '@element-plus/utils/with-install'
 
 import Message from './src/message-method'
 
-export const ElMessage = withInstallFunction(Message, '$message')
+export const ElMessage = withFunctionInstall(Message, '$message')
 export default ElMessage
 
```

Comment:
Why are we making irrelevant changes here? Seems like this was not related to `menu` component

Reply:
Sorry. This branch is after the message branch, I will rebase it.